x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/components/phases/shared_fields/data_tier_allocation_field/components/default_to_data_tiers_notice.tsx

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { FunctionComponent } from 'react';
+import { EuiCallOut } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import { PhaseWithAllocation } from '../../../../../../../../../common/types';
+
+const i18nTexts = {
+  title: {
+    warm: i18n.translate(
+      'xpack.indexLifecycleMgmt.warmPhase.dataTier.defaultAllocationNotAvailableTitle',
+      { defaultMessage: 'Indices will be allocated to the warm tier' }
```

Comment:
The update callout content will be:

> **Move data to warm nodes**
> Define custom node attributes in elasticsearch.yml to allocate indices based on these attributes instead.

In this particular situation, the user's selected "Custom", and probably intends to use custom node attributes to control allocation behavior. However, because they haven't defined any custom node attributes, allocation will behave unexpectedly, depending on how the cluster is configured.

"Move data to warm nodes" sounds imperative to me, as if the user intentionally chooses this option in order to allocate to warm nodes. But the scenario is more like "You probably didn't intend this, so we just wanted to be clear about what's going to happen."

Is this making any sense @debadair? Does it change the way you're seeing this copy?